src/tdastro/astro_utils/opsim.py

```diff
@@ -291,3 +293,39 @@ def get_observations(self, query_ra, query_dec, radius, cols=None):
                 raise KeyError(f"Unrecognized column name {table_col}")
             results[col] = self.table[table_col][neighbors].to_numpy()
         return results
+
+    def flux_err_point_source(self, flux, index):
+        """Simulate photon noise for a point source observation
```

Comment:
I was a little confused by the comment when I first read this, because I thought it would be sampling from the noise distribution. Maybe we should say something like "Compute the distribution parameters of photon noise..."